datadump/pre-population/cve-ids-range.json

```diff
@@ -0,0 +1,347 @@
+            "start": 10000,
+            "end": 10000
+        },
+        "general": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 50000000
+        }
+    }
+  },
+  {
+    "cve_year": 2011,
+    "ranges": {
+        "priority": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 10000
+        },
+        "general": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 50000000
+        }
+    }
+  },
+  {
+    "cve_year": 2012,
+    "ranges": {
+        "priority": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 10000
+        },
+        "general": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 50000000
+        }
+    }
+  },
+  {
+    "cve_year": 2013,
+    "ranges": {
+        "priority": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 10000
+        },
+        "general": {
+            "top_id": 10000,
+            "start": 10000,
+            "end": 50000000
+        }
+    }
+  },
+  {
+    "cve_year": 2014,
+    "ranges": {
+        "priority": {
+            "top_id": 150000,
+            "start": 150000,
+            "end": 150000
+        },
+        "general": {
+            "top_id": 150000,
+            "start": 150000,
+            "end": 450000
+        }
+    }
+  },
+  {
+    "cve_year": 2015,
+    "ranges": {
+        "priority": {
+            "top_id": 20000,
+            "start": 20000,
+            "end": 20000
+        },
+        "general": {
+            "top_id": 20000,
+            "start": 20000,
+            "end": 999999
+        }
+    }
+  },
+  {
+    "cve_year": 2016,
+    "ranges": {
+        "priority": {
+            "top_id": 20000,
+            "start": 20000,
+            "end": 20000
+        },
+        "general": {
+            "top_id": 20000,
+            "start": 20000,
+            "end": 550000
+        }
+    }
+  },
+  {
+    "cve_year": 2017,
+    "ranges": {
+        "priority": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 30000
+        },
+        "general": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 999999
+        }
+    }
+  },
+  {
+    "cve_year": 2018,
+    "ranges": {
+        "priority": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 30000
+        },
+        "general": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 550000
+        }
+    }
+  },
+  {
+    "cve_year": 2019,
+    "ranges": {
+        "priority": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 30000
+        },
+        "general": {
+            "top_id": 30000,
+            "start": 30000,
+            "end": 999999
+        }
+    }
+  },
+  {
+    "cve_year": 2020,
+    "ranges": {
+        "priority": {
+            "top_id": 0,
```

Comment:
Oh yeah, priorities won't exist for 2020 either. Only new years with IDR will have priority ranges